doc/using_tf.rst

```diff
@@ -725,6 +725,8 @@ Create Python Scripts for Custom Input and Output Formats
 
 You can add your customized Python code to process your input and output data:
 
+This customized Python code must be named ``inference.py`` and specified through the ``entry_point`` parameter:
+
```

Comment:
can you combine these into one paragraph? (and change the first colon to a period)